src/lib/core/common-behaviors/common-module.ts

```diff
@@ -104,6 +107,18 @@ export class MatCommonModule {
     this._document.body.removeChild(testElement);
   }
 
+  /** Checks whether the material version matches the cdk version */
+  private _checkCdkVersionMatch(): void {
+    if (VERSION.major !== CDK_VERSION.major && VERSION.minor !== CDK_VERSION.minor) {
```

Comment:
I actually want to go so far as to say that the versions have to _exactly_ match. You could do this by comparing `VERSION.full !== CDK_VERSION.full`

Reply:
@jelbourn I updated the PR to compare the full version. could you please tell me how you want me to proceed concerning the test-build issues (see my comment above: https://github.com/angular/material2/pull/15395#issuecomment-470112826)?

Reply:
I exported an additional symbol `export const ɵVERSION` in the `@angular/cdk/bidi` module and added another non-exportet `VERSION` symbol in the `common-module.ts`, otherwise it would be necessary to adapt various build configs to allow accessing the `VERSION` symbol from `@angular/material` and `@angular/cdk`.
The build and tests now work without failures.

Reply:
This shouldn't be necessary. @devversion could you advise on the build issue here?

Reply:
@macjohnny Yeah. You should be able to import the version from `@angular/material` and `@angular/cdk`. In order to allow this, we need to adjust the tsconfig path mapping (like you already did with your initial commit).

Also for Bazel, you need to update the `ng_module` definition [here](https://github.com/angular/material2/blob/master/src/lib/core/BUILD.bazel#L26). Just add `//src/cdk` there (before `//src/cdk/a11y`).

Though it will be a bit more difficult for the version from Material. We don't want `core` to depend on the `@angular/material` primary entry-point which transitively depends on all Material secondary entry-points (e.g. `slide-toggle`, `list` etc.)

Maybe the best would be to just use the version placeholder in `core` (like it's done right now) with a proper comment. Eventually we can remove this once the Material primary entry-point no longer re-exports all secondary entry-points (see how the CDK entry-point works)

Reply:
@devversion @jelbourn I updated the PR according to your suggestions. the build now works without additionally exporting the `VERSION` symbol. could you have a look at it again?